releasenotes.txt

```diff
@@ -44,3 +44,6 @@ Other:
 - Markup files will now be processed after the regular source files when using multiple threads/processes (some issues remain - see Trac #12167 for details).
 - Added file name to ValueFlow "--debug" output.
 - Fixed build when using "clang-cl" in CMake.
+
+Safety critical fixes:
+- Added "--safety" option. It makes Cppcheck more strict about critical errors.
```

Comment:
That should not be in the release notes as it is not documented.

If we make this official we cannot change this without a transition period and makes it harder to adjust it later on.

Reply:
I agree. hmm.. I will write this in the proper Cppcheck Premium release notes instead.

I am trying to satisfy conflicting requirements. I can understand that from your point of view this is a quick hack that is not needed. :-(
For Cppcheck Premium the safety certification is very important so we need to satisfy certain requirements, but at the same time I really don't want to make it difficult for existing open source users. Keeping existing open source users happy is also business critical imho.